scripts/parallel-test-runner.sh

```
@@ -0,0 +1,124 @@
+  "e2e-webkit:npx playwright test --project=webkit"
+  "visual:npm run test:visual"
+  "fumadocs:npm run test:fumadocs"
+  "performance:npm run test:perf"
+)
+
+# Run tests in parallel
+echo -e "\n${YELLOW}Running ${#SUITES[@]} test suites in parallel...${NC}\n"
+
+# Use GNU parallel if available, otherwise fall back to background processes
+if command -v parallel &> /dev/null; then
+  printf '%s\n' "${SUITES[@]}" | parallel -j 4 --colsep ':' run_suite {1} {2}
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)